extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSSchemaUtils.scala

```diff
@@ -20,51 +20,45 @@ package org.apache.kyuubi.spark.connector.tpcds
 import scala.collection.JavaConverters._
 
 import io.trino.tpcds.Table
+import io.trino.tpcds.Table._
+import io.trino.tpcds.column._
 import io.trino.tpcds.generator._
 
-object TPCDSTableUtils {
```

Comment:
Split the `TPCDSTableUtils` into `TPCDSSchemaUtils` and `TPCDSStatisticsUtils`, and use ENUM instead of String as much as possible.
cc @wForget

Reply:
LGTM